lib/flow.js

```diff
@@ -2,8 +2,19 @@
 
 module.exports = (api) => {
 
+  api.metasync.flow = (
+    // Create a Flow instance
```

Comment:
That's a terminological question, flow instance is an instance of function composed from input syntax but I'll change comments